Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -36,7 +36,7 @@ extension Lambda {
         /// `ByteBufferAllocator` to allocate `ByteBuffer`
         public let allocator: ByteBufferAllocator
 
-        internal init(logger: Logger, eventLoop: EventLoop, allocator: ByteBufferAllocator) {
+        public init(logger: Logger, eventLoop: EventLoop, allocator: ByteBufferAllocator) {
```

Comment:
@tomerd We probably need to discuss this one! Maybe we want to add a __forTest_Only() helper method instead of the public initializer.

Reply:
yes this not ideal. why is it needed?